### ISSUE TITLE
unknown command: goog/cdp/execute

### ISSUE THREAD

**Issue by trev** [SeleniumHQ/selenium#6632]
## Meta -
OS:  OSX
Selenium Version:  selenium-webdriver (3.141.0)
Browser:  Chrome

Browser Version:  
Looks like it's using the latest Chrome release from a Debian package: https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/ruby/images/2.5.0-stretch/node-browsers/Dockerfile#L62-L76

Assuming that's 72.0.3602.2

## Expected Behavior -
To successfully set the `driver.browser.download_path`

## Actual Behavior -
Since upgrading from 3.14.0 to 3.141.0 we get the following failure on CircleCI with the `circleci/ruby:2.5.0-node-browsers` docker image. When we rollback to 3.14.0 all is good again.

```
Selenium::WebDriver::Error::WebDriverError:
       unexpected response, code=404, content-type="text/plain"
       unknown command: session/33427b05393b6b34815b0a386f36e05d/goog/cdp/execute
```

## Steps to reproduce -

Run on CircleCI with `circleci/ruby:2.5.0-node-browsers`.

```
From: /home/circleci/app/spec/support/capybara.rb @ line 12 :

     9:   capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(chromeOptions: {args: args})
    10:   driver = Capybara::Selenium::Driver.new(app, browser: :chrome, desired_capabilities: capabilities)
    11:   binding.pry
 => 12:   driver.browser.download_path = DownloadHelper::PATH.to_s
    13:   driver
    14: end
    15:
    16: Capybara.javascript_driver = :chrome
    17: Capybara.server = :puma, {Silent: true}

[1] pry(main)> DownloadHelper::PATH.to_s
=> "/home/circleci/app/tmp/downloads/0"

[2] pry(main)> driver.browser.download_path = DownloadHelper::PATH.to_s

Selenium::WebDriver::Error::WebDriverError: unexpected response, code=404, content-type="text/plain"
unknown command: session/2215cc5b53c0df4c7a20b977a8e0f9aa/goog/cdp/execute
from /home/circleci/app/vendor/bundle/ruby/2.5.0/gems/selenium-webdriver-3.141.0/lib/selenium/webdriver/remote/http/common.rb:91:in `create_response'

[3] pry(main)> driver.browser
=> #<Selenium::WebDriver::Chrome::Driver:0x262164beef79bb4c browser=:chrome>
```

**Comment by p0deje** [comment]
ChromeDriver changed API for sending custom commands in v2.40 https://bugs.chromium.org/p/chromedriver/issues/detail?id=2307. In Selenium 3.141.0 we update client bindings to also use a new API, meaning you need at least ChromeDriver 2.40 to work with it. Can you verify its version?

P.S. This was mentioned in [changelog](https://github.com/SeleniumHQ/selenium/blob/master/rb/CHANGES#L12-L13)

**Comment by trev** [comment]
@p0deje Thanks for this. It does seem like it's older than v.2.40:

```
circleci@f471ca203720:~/app$ chromedriver --version
ChromeDriver 2.37.544315 (730aa6a5fdba159ac9f4c1e8cbc59bf1b5ce12b7)
circleci@f471ca203720:~/sapp$ google-chrome --version
Google Chrome 64.0.3282.186
```

The interesting thing is that [`curl --location --fail --retry 3 http://chromedriver.storage.googleapis.com/LATEST_RELEASE`](https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/ruby/images/2.5.0-stretch/node-browsers/Dockerfile#L69) returns `2.43` yet it's `2.37.544315` that's being used.

Actually upon further inspection, I guess it's because this image uses `Chrome 64` which relates to `ChromeDriver 2.37`.

So basically, I need a fresher image. Thanks for the input, much appreciated.

**Comment by trev** [comment]
Confirmed that this works when using an a Docker image that users a latest stable release of Chrome: https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/ruby/images/2.5.3-stretch/node-browsers/Dockerfile#L50-L64.

Thanks again @p0deje
